src/data/get_data.py

```diff
@@ -18,6 +19,9 @@ def main():
     test_data = sys.argv[2]
     val_data = sys.argv[3]
 
+    gdown.download_folder('https://drive.google.com/drive/folders/1_NobSEMZS8jogSAEZ9ZLBUTemPiASJRg',
+                          output="data", quiet=False)

```

Comment:
I don't know if this is the correct solution as they are already downloaded with the dvc pull. And with a cleared cache it worked for everyone.

Reply:
Now it is a part of the pipeline though and not just pulled. Also it didn't work for some by just clearing the cache, fetching each individual file was needed which is a hassle for people reviewing.

Reply:
I'll approve it as it does work, but I still think that this is not the intended way of resolving the problem